integration_test.go

```diff
@@ -1,5 +1,3 @@
-// +build integration
```

Comment:
I believe this is to prevent trying to run the tests in "regular" CI which doesn't provide any credentials.

Reply:
I'll re-add that